### PR TITLE
[HttpFoundation] Added the ability of using BinaryFileResponse with stream wrappers

### PR DESCRIPTION
**Pull request by jakzal**
| Q | A |
| --- | --- |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | #12990 #13187 |
| License | MIT |
| Doc PR | ~ |

### CODE REVIEW

File: src/Symfony/Component/HttpFoundation/BinaryFileResponse.php [hunk 1]
```diff
@@ -192,6 +192,10 @@ public function prepare(Request $request)
             // Use X-Sendfile, do not send any content.
             $type = $request->headers->get('X-Sendfile-Type');
             $path = $this->file->getRealPath();
+            // Fall back to scheme://path for stream wrapped locations.
+            if (empty($path)) {
```
Comment (aitboudad) [inline src/Symfony/Component/HttpFoundation/BinaryFileResponse.php]: just check if is equal `false`

**Comment by dunglas** [comment]
:+1: 

Status: Reviewed

**Comment by nicolas-grekas** [comment]
Thank you @jakzal.

**Comment by nicolas-grekas** [comment]
Thank you @Sander-Toonen

**Pull request merged**